Extension/src/common.ts

```diff
@@ -172,9 +172,9 @@ export function getVcpkgPathDescriptorFile(): string {
     }
 }
 
-let vcpkgRoot: string;
+let vcpkgRoot: string | undefined;
 export function getVcpkgRoot(): string {
-    if (!vcpkgRoot) {
+    if (!vcpkgRoot && vcpkgRoot !== "") {
```

Comment:
Yeah, I noticed that when I code review...I was thinking about asking what that was for :)

Reply:
It looks like it's using a value of "" to indicate that it has attempted and failed, so does not attempt again.